js/index.js

```diff
@@ -0,0 +1,63 @@
+$(document).ready(function() {
+
+    // adds operand or operator to screen
+
+    $('.buttons > span').click(function() {
+        if (isDone) {
+            reset()
+        }
+        $('#screen').append($(this).text())
+    });
+
+    $('#zero').click(function() {
+        if (isDone) {
+            reset()
+        }
+        $('#screen').append($(this).text())
+    });
+
```

Comment:
DRY- don't repeat yourself. If you're writing identical code twice, you can almost always combine it into one